packages/cli/src/commands/link/link.ts

```diff
@@ -32,6 +33,7 @@ async function link(
   ctx: Config,
   opts: FlagsType,
 ) {
+  printDeprecationWarning('link');
```

Comment:
`link` command itself links assets by default, which is still supported – maybe we should rename it to `link-assets`? Not sure what's best here. The thing is that `link [packageName]` and `link --all` get deprecated and we need a clear path forward

Reply:
Oh righty. Forgot about the assets.